src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineScaleSetExtensionTests.ps1

```diff
@@ -175,6 +175,29 @@ function Test-DisableVirtualMachineScaleSetDiskEncryption
     }
 }
 
+<#
+.SYNOPSIS
+Test Virtual Machine Scale Set Disk Encryption Extension
+#>
+function Test-DisableVirtualMachineScaleSetDiskEncryption2
+{
+    try
+    {
+        # Common
+        $loc = 'westcentralus';
+        $rgname = "hyleevmssdetest2"
+        $vmssName = 'vmss' + $rgname;
+
+        $result = Disable-AzureRmVmssDiskEncryption -ResourceGroupName $rgname -VMScaleSetName $vmssName -Force;
+        $result_string = $result | Out-String;
```

Comment:
@hyonholee are we checking anything with this resulting string, or are we just ensuring that an error is not thrown when the disk encryption extension is not the first extension?

Reply:
@cormacpayne  the test runs successfully if the error is not thrown when the disk encryption is not the first extension.  Without the fix, this test fails with an exception thrown.

Reply:
@hyonholee thanks for the clarification, LGTM